### PR TITLE
Grafana intermittent fix

### PR DESCRIPTION
**Pull request by hmuthusamy**
*Issue #, if available:*

*Description of changes:*
Fix role issues on Grafana

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: platform/infra/terraform/setup-keycloak.sh [hunk 1]
```diff
@@ -131,6 +131,7 @@ while true; do\n
     ./kcadm.sh create users -r $KEYCLOAK_REALM -f /tmp/admin.json --config /tmp/kcadm.config\n
     ./kcadm.sh create users -r $KEYCLOAK_REALM -f /tmp/editor.json --config /tmp/kcadm.config\n
     ./kcadm.sh create users -r $KEYCLOAK_REALM -f /tmp/viewer.json --config /tmp/kcadm.config\n
+    ./kcadm.sh add-roles --uusername user1 --rolename "grafana-admin" -r $KEYCLOAK_REALM --config /tmp/kcadm.config\n
```
Comment (elamaran11) [inline platform/infra/terraform/setup-keycloak.sh:134]: Is `uusername` correct?

**Review by elamaran11 — CHANGES_REQUESTED** [review]
Check my comment

File: platform/infra/terraform/setup-keycloak.sh (hunk 1 above)
Comment (shapirov103) [inline platform/infra/terraform/setup-keycloak.sh:134]: `--uusername` just making sure it is correct (double u)

**Review by shapirov103 — COMMENTED** [review]
see my comment

File: platform/infra/terraform/setup-keycloak.sh (hunk 1 above)
Comment (hmuthusamy) [reply platform/infra/terraform/setup-keycloak.sh:134]: yes. thats the API

**Review by elamaran11 — APPROVED** [review]
LGTM

**Review by shapirov103 — APPROVED** [review]

**Pull request merged**